src/main/java/com/sonyericsson/jenkins/plugins/bfa/GerritMessageProviderExtension.java

```diff
@@ -98,5 +98,12 @@ private void addFailureCausesFromData(StringBuilder message, FailureCauseDisplay
             .append(displayData.getLinks().getBuildUrl())
             .append(" )");
         }
+
+        if (displayData.getDownstreamFailureCauses().isEmpty() && displayData.getFoundFailureCauses().isEmpty()) {
+            message.append("Unknown (")
```

Comment:
If we are going to use the standard value, then use the configured noCausesMessage, not the default one.
I could see a point in shortening it though, since "please add a suitable cause for it" isn't really helpful from Gerrit, you just want to know what went wrong. 
Maybe just "No problems were identified", then it at least looks like the default message that people are used to.